documentation/bolt_installing_modules.md

```diff
@@ -78,6 +77,11 @@ This list of packaged modules is available in a
 Bolt repository. The modules and supporting documentation are publicly available
 on the [Puppet Forge](https://forge.puppet.com/).
 
+If you need to use a specific version of a packaged module, you can override the
+packaged version by installing the module into your `modules/` directory.
```

Comment:
I'm not sure if this is worth detailing, or if it's TMI, but modules installed anywhere on the modulepath will override packaged modules. Where I think that might get confusing is if the user *doesn't* have `modules/` on the modulepath (for example they've configured their modulepath and don't have `<project>/modules` there) then the installed module won't get loaded at all and won't override packaged modules.